docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -126,19 +206,21 @@ public class NoOpSingleStringFunction implements StringFunction {
     }
 }
 ----
-
-Quarkus also provides support for injecting configuration values using Spring's `@Value` annotation.
-To see that in action, first edit the `src/main/resources/application.properties` with the following content:
-
-[source,properties]
++
+//. Quarkus also provides support for injecting configuration values using Spring's `@Value` annotation.
+. In {ProductName} applications, you can also use the `@Value` annotations to inject values of configuration properties into beans:
```

Comment:
Why need for saying "applications" here ? It's "heavy" when Users don't necessarily make full applications but just demos, samples, micro services or server less functions. Why not just "with Quarkus,"

Reply:
The original idea behind the change is that enterprise application developers (as our default user persona for product docs) would predominantly be looking into developing full applications. That being said, you make a compelling point. I think it might be worthwhile to make the wording less explicit here to allow for a broader spectrum of potential use cases, not just full apps.